roles/kube_lvm_storage/defaults/main.yml

```diff
@@ -8,44 +8,15 @@ storage_addon_dir: '{{ kube_config_dir }}/addons/storage_lvm'
 
 # Set the storage class setup on kubernetes node
 metal_k8s_storage_class:
```

Comment:
We use `metalk8s_elasticsearch_enabled`, so I guess for consistency sake it shall be `metalk8s_*` everywhere in this PR as well...

Reply:
That's what I'm doing :)